blog/2024-12-15-openhab-4-3-release.md

```diff
@@ -80,6 +80,14 @@ For example, the route `/page/tabs/0` will open the first tab of the page `tabs`
 
 Please refer to the [documentation](/docs/ui/tabbed-pages.html#navigating-tabs) for more information.
 
+### Dirty / Modified Indicator
+
+When you modify a configuration/setting, a ● symbol is now shown next to the page title to indicate that there are unsaved changes and the configuration page becomes "dirty":
+
+<p align="center"><img style="max-width: 70%; max-height: 1000px;" src="/uploads/2024-12-15-openhab-4-3-release/dirty-indicator.34733bcb.png"/></p>
```

Comment:
I don‘t think setting max height is needed due to the landscape format of the image:

```suggestion
<p align="center"><img style="max-width: 70%;" src="/uploads/2024-12-15-openhab-4-3-release/dirty-indicator.34733bcb.png"/></p>
```